davitpy/pydarn/radar/radFov.py

```diff
@@ -428,8 +467,9 @@ def __str__(self):
 
 # *************************************************************
 # *************************************************************
```

Comment:
Removed camelCase for all non-keyword variables, as well as moving the standard model to a separate routine, adding the Chisham model, fixing the altitude implementation, and fixing the while-loop to make in possible to return locations for 1.5-hop backscatter